gr-utils/python/modtool/core/info.py

```diff
@@ -57,6 +57,8 @@ def run(self):
                 os.path.isdir(os.path.join('include', 'gnuradio', mod_info['modname']))
                 ):
             self.info['version'] = '37'
+        if not os.path.isfile(os.path.join('cmake', 'Modules', 'FindCppUnit.cmake')):
```

Comment:
Not sure if this is the best way to do this.... but OK.

Reply:
It was the code in place, but in a different place - went with the quick fix.  I can make this more robust if you think it is necessary - and ideally a version number would be dumped into the module directory when the modtool is run.  Would you support this as a method going forward, to get a more specific version as what version of the modtool created the module?

Reply:
That would be a good future change, but for now, this is fine.